aisynphys/ui/dashboard.py

```diff
@@ -633,12 +633,16 @@ def check(self):
             search_paths = [self.primary_path, self.archive_path, self.nas_path]
             submitted = False
             connections = None
+            ignore = False
             for path in search_paths:
                 if path is None:
                     continue
                 yml_file = os.path.join(path, 'pipettes.yml')
                 if os.path.isfile(yml_file):
                     submitted = True
```

Comment:
This seems to change the original logic.. shouldn't there be a `break` here, or was that intentional?

Reply:
mm yeah the break is now after the ignore file check, so I guess they should be 2 separate loops?

Reply:
I think you just need another `break` at this line.

Reply:
won't that exit the for loop potentially before an ignore file is found?